x-pack/plugins/observability_solution/apm/public/components/routing/templates/apm_main_template/inventory_callout.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiLink, EuiToolTip, EuiButtonIcon } from '@elastic/eui';
+import { TechnicalPreviewBadge } from '@kbn/observability-shared-plugin/public';
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { ENTITY_INVENTORY_LOCATOR_ID } from '@kbn/observability-shared-plugin/common';
+import { SerializableRecord } from '@kbn/utility-types';
+import { useApmPluginContext } from '../../../../context/apm_plugin/use_apm_plugin_context';
+
+export interface InventoryCalloutProps {
+  onDissmiss: () => void;
+}
+
+export function InventoryCallout({ onDissmiss }: InventoryCalloutProps) {
+  const { share } = useApmPluginContext();
+  const inventoryLocator = share.url.locators.get<SerializableRecord>(ENTITY_INVENTORY_LOCATOR_ID);
+
+  return (
+    <EuiFlexGroup direction="row" alignItems="center" gutterSize="xs">
+      <EuiFlexItem grow={false}>
+        <TechnicalPreviewBadge icon="beaker" style={{ verticalAlign: 'middle' }} />
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>
+        <EuiLink data-test-subj="inventoryCalloutLink" href={inventoryLocator?.useUrl({})}>
```

Comment:
I know we're not very consistent with `data-test-subj` but it's always good idea to have the plugin name in the start. 
```suggestion
        <EuiLink data-test-subj="apmInventoryCalloutLink" href={inventoryLocator?.useUrl({})}>
```